.versionpress/schema.yml

```diff
@@ -0,0 +1,15 @@
+pods_relationship:
+  table: podsrel
+  id: id
+  references:
+    pod_id: post
+    field_id: post
+    item_id: post
```

Comment:
Right, thank you! I did not realize that, I quickly started with extended post. We will have to use custom function, similar example is used in VP core for `_menu_item_object_id` ([here](https://github.com/versionpress/versionpress/blob/d3d0c984dff174d70ca67555a0b0b9ca325958f3/plugins/versionpress/.versionpress/schema.yml#L30)). Pods API will probably have similar function to get item type for postmeta value? I will try to look at it.